specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/StaticSites.json

```diff
@@ -2688,6 +2688,127 @@
     }
   },
   "definitions": {
+    "ArmPlan": {
+      "description": "The plan object in Azure Resource Manager, represents a marketplace plan.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name.",
+          "type": "string"
+        },
+        "publisher": {
+          "description": "The publisher.",
+          "type": "string"
+        },
+        "product": {
+          "description": "The product.",
+          "type": "string"
+        },
+        "promotionCode": {
+          "description": "The promotion code.",
+          "type": "string"
+        },
+        "version": {
+          "description": "Version of product.",
+          "type": "string"
+        }
+      }
+    },
+    "RemotePrivateEndpointConnection": {
+      "description": "A remote private endpoint connection",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "RemotePrivateEndpointConnection resource specific properties",
+          "properties": {
```

Comment:
>properties [](start = 11, length = 10)

are you sure you need properties in the 2 lines above as well?

Reply:
I still think you have one too many 'properties' in this schema definition

---
In reply to: [607452734](https://github.com/Azure/azure-rest-api-specs/pull/13768#discussion_r607452734) [](ancestors = 607452734)